CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Changed android api target level to 30
 - Subject of error reports now start with version number
+- Enable automatic polling per default on iOS devices (to fix missing push challenges)
```

Comment:
i think we should do this for android devices too.

Reply:
> While we (you) are at it, can you lower the polling interval to something like 5 or 3s? Polling is only done if the app is in foreground right?

That's right. I changed the interval to 3 seconds.